packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -142,12 +170,7 @@ const helpers = {
         return pipe(
           httpContent,
           Option.fold(
-            () => {
-              logger.warn(`Unable to find a content for ${mediaTypes}`);
-              return Either.left<Error, IHttpNegotiationResult>(
-                ProblemJsonError.fromTemplate(NOT_ACCEPTABLE, `Unable to find content for ${mediaTypes}`),
-              );
-            },
+            () => whenNoContent(logger, response, headers || [], mediaTypes),
```

Comment:
You should also apply the support for empty content here: https://github.com/stoplightio/prism/blob/master/packages/http/src/mocker/negotiator/NegotiatorHelpers.ts#L316

It will let you "hit two birds with one stone". This [PR](https://github.com/stoplightio/prism/pull/609) introduces empty response for invalid requests, however you solution looks more generic.

If you feel that this is not in scope I'll create new PR based on yours.

Reply:
thanks @StefanDywersant for pointing this out, will take care of this in this PR